### ISSUE TITLE
Install error

### ISSUE THREAD

**Issue by yomono** [RestyaPlatform/board#1299]
Hi.
Trying to install restyaboard.sh, i got this error:
+ psql -U postgres -c \q
psql: could not connect to server: No existe el archivo o el directorio
        Is the server running locally and accepting
        connections on Unix domain socket "/var/run/postgresql/.s.PGSQL.5432"?
+ [ 2 != 0 ]
+ echo PostgreSQL Changing the permission failed with error code 34
PostgreSQL Changing the permission failed with error code 34

Can anybody help me?
Thank you in advance

**Comment by lmarikannan** [comment]
@yomono We checked about this, there is an issue in postgresql database connection. For more information, can you please share the install log (remove sensitive details)?

**Comment by yomono** [comment]
Sorry, i totally forgot to upload the install log.
Anyway, it appeared to be a previous postgres install that caused the problem. 
I just reinstalled Ubuntu and started it over again, and it is working now.
I had to change apache port to 81 because the nginx process was preventing apache to start (weird.. i mean, resyaboard.sh installs both, but both uses 80 port, so how it works?).

Now im facing another issue: IMAP its not working (i want to create tasks from mails). Should i open another issue?

Thank you in advance, and sorry for my english, im from argentina
